waffle_configs/bedrock-prod.env

```diff
@@ -36,9 +36,9 @@ SWITCH_WELCOME_8_EXPERIMENT=on
 SWITCH_WHATSNEW_FIREFOX_98_TURNING_RED=off
 SWITCH_VPN_FUNNEL_OPT_SUBSCRIPTION_URL=on
 SWITCH_BROWSER_MR2_PROMO=on
-SWITCH_HOMEPAGE_MR2_PROMO=on
```

Comment:
This switch is still needed for the global home page. We'll need to remove it from the bedrock template before we can safely remove it from the config.
https://github.com/mozilla/bedrock/blob/main/bedrock/mozorg/templates/mozorg/home/home.html#L15